subworkflows/ebi-metagenomics/read_qc/main.nf

```diff
@@ -0,0 +1,30 @@
+
+include { FASTP      } from '../../../modules/ebi-metagenomics/fastp/main'
+include { SEQTK_SEQ     } from '../../../modules/ebi-metagenomics/seqtk/seq/main'
+
+workflow  READ_QC {
+
+    take:
+    ch_reads // channel: [ val(meta), [ fastq ] ]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    save_trimmed_fail = true
```

Comment:
I would recommend to move `save_trimmed_fail` and `save_merged` to config and use as `params.save_trimmed_fail` and `params.save_merged` 
like https://github.com/nf-core/mag/blob/master/subworkflows/local/binning.nf#L69 
I took this idea from one of nf-core subwfs

Reply:
Done! Thanks for reviewing 😄